themes/delphi/layouts/partials/delphi-text-logo.html

```diff
@@ -0,0 +1 @@
+<a class="delphi-text-logo" href="https://delphi.cmu.edu/">DELPHI</a>
```

Comment:
see https://gohugo.io/functions/relref/ to create references in hugo

Reply:
How do you use this to link to the homepage from anywhere on the site? Relref seems to be relative to where the user is on the site? Definitely not understanding something here.